compressor/Compressor.go

```diff
@@ -0,0 +1,6 @@
+package compressor
+
+//Compressor interface for sound compressor's
+type Compressor interface {
```

Comment:
In Go interfaces are usually defined in user packages. So `Mixer` should define what it accepts as a `Compressor`.